# Ego vehicle
conformal_lattice_planner/Vehicle ego
# Agent vehicles
conformal_lattice_planner/Vehicle[] agents
